packages/TUNZIP.yaml

```
@@ -0,0 +1,34 @@
+---
+name: 'TUNZIP'
+version: '0.91'
+release: 1
+summary: 'TuNzIp, TNIs zip decompression tool decompression tool'
+author: 'Patriek Lesparre'
+package_author: 'Fubukimaru'
+license: 'Freeware'
+category: 'Tools'
```

Being a file decompression tool, maybe the `Files` section would be more correct?

```suggestion
category: 'Files'
```